src/Apps/Artwork/Components/__tests__/PricingContext.test.tsx

```diff
@@ -14,7 +14,8 @@ jest.unmock("react-tracking")
 jest.unmock("react-relay")
 
 const mockPricingContext = {
-  filterDescription: `small mocks by David Sheldrick`,
+  appliedFiltersDisplay: "Price ranges of small mocks by David Sheldrick",
+  filterDescription: `deprecated field`,
```

Comment:
Do we need this here?

Reply:
since technically is still not removed i left it there, we can do a cleanup after the 🚋